code/core/src/components/components/Colors/colorpalette.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, ColorPalette, ColorItem } from '@storybook/blocks';
 
-import { themes, ThemeProvider, convert, ensure } from '@storybook/theming';
+import { themes, ThemeProvider, convert, ensure } from '@storybook/core/theming';
```

Comment:
Should `@storybook/core` ever been a package publicly available, if we plan to merge it into `storybook`? `storybook` already provides `storybook/theming`. Shouldn't this be used instead?

Reply:
I have just seen this `mdx` file, and I thought it was user-facing. My bad if it is just internal documentation.